pkg/collector/corechecks/snmp/internal/checkconfig/config.go

```diff
@@ -293,7 +293,7 @@ func (c *CheckConfig) GetNetworkTags() []string {
 // warning: changing getDeviceIDTags logic might lead to different deviceID
 func (c *CheckConfig) getDeviceIDTags() []string {
 	tags := []string{deviceNamespaceTagKey + ":" + c.Namespace, deviceIPTagKey + ":" + c.IPAddress}
-	sort.Strings(tags)
+	sortutil.UniqInPlace(tags)
```

Comment:
Internally, `sortutil.UniqInPlace` sorts tags and removes duplicates which should be done for a slice of tags.
The only difference is that a new array is not allocated.